libs/logger/DataActionLogger.php

```diff
@@ -1,9 +1,8 @@
 <?php
 
 namespace Leaf;
+class DataActionLogger{
 

```

Comment:
for classes the { is supposed to be on the next line by itself. According to PSR12 and for the rest of the files.